scripts/pregenerate/__init__.py

```diff
@@ -55,6 +55,8 @@ class TargetFilter:
     # If non-empty only the given paths will be code-generated
     path_glob: List[str] = field(default_factory=list)
 
+    cpp_only: bool = False
```

Comment:
other members have comments explaining things. This needs one too.

Reply:
Filed https://github.com/project-chip/connectedhomeip/issues/24553 to track this.